### ISSUE TITLE
Using with ui-validate causes chosen to break when value is invalid

### ISSUE THREAD

**Issue by adrianboimvaser** [leocaseiro/angular-chosen#35]
I will provide more details and a JSFidde later.

**Comment by leocaseiro** [comment]
Sorry the delay, I've just took this project that was abandoned (read #162).

The issue with ui-validate is not only with angular-chosen, but also with any multiple select.

If you remove the chosen directive from your select, it's still having an error:
"Advanced Criteria Required when Multiple Markets"

I found a solution that might be a fix for you https://github.com/angular-ui/ui-select/issues/914

**Comment by chexxor** [comment]
I haven't used this in awhile, so I've no stake in this bug anymore. Feel free to close if the issue creator doesn't mind.
I appreciate your stepping up as a new maintainer, though! Good luck and have fun!

**Comment by leocaseiro** [comment]
Thanks for your feedback @chexxor it's a shame that almost 2 years this still an issue. I'll have a deep look into ui-validate and fix this in your plunkr before close the issue.

**Comment by leocaseiro** [comment]
I've fixed the same way I'd fix with ng-required #87  or required #169 only with CSS.

```css
.localytics-chosen {
  display: block !important;
  position:absolute;
  clip:rect(0,0,0,0);
  height: 29px;
}
```

Your plunkr up to date with ui-validate working
http://plnkr.co/edit/QMEeIb?p=preview

**Comment by leocaseiro** [comment]
:beers: Thanks!
